recipes/dionysus/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "dionysus" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "78c24da06fd273f0103ad4cb86331f5f4518d7f0445e2cecd0e325ca1a2d011f" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  script: python setup.py install --single-version-externally-managed --record record.txt  # [not linux]
+  script: >
+    CC=clang CXX=clang++ CXXFLAGS=-stdlib=libc++ LDFLAGS=-lc++abi
+    python setup.py install --single-version-externally-managed --record record.txt  # [linux]
```

Comment:
Having multiple scripts gives a warning when reading as yaml. You can do

```
script: 
  - python setup.py install --single-version-externally-managed --record record.txt  # [not linux]
  - CC=clang CXX=clang++ CXXFLAGS=-stdlib=libc++ LDFLAGS=-lc++abi python setup.py install --single-version-externally-managed --record record.txt  # [linux]
```

Reply:
@isuruf My suggestion in https://github.com/conda-forge/staged-recipes/pull/3976#issuecomment-330100756 should be okay too, though a little unusual, right?

Reply:
It's unsual, but if it works, sure.